CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 ### Removed
 
 - [#5824](https://github.com/thanos-io/thanos/pull/5824) Mixin: Remove noisy `ThanosReceiveTrafficBelowThreshold` alert.
+- [#5819](https://github.com/thanos-io/thanos/pull/5819) Store: Add a few objectives for Store's data touched/fetched amount and sizes. They are: 50, 95, and 99 quantiles.
```

Comment:
I guess this was accidentally put into removed section instead of added?

Reply:
Oh, my bad, @matej-g. Fixed in https://github.com/thanos-io/thanos/pull/5919.